src/spritekit.cs

```diff
@@ -299,14 +299,18 @@ partial interface SKNode : NSCoding, NSCopying {
 		[Export ("attributeValues", ArgumentSemantic.Copy)]
 		NSDictionary<NSString, SKAttributeValue> AttributeValues { get; set; }
 
+#if !XAMCORE_4_0
+		[Obsolete ("Please use the subtype's method")]
```

Comment:
I think we can remove the [Obsolete] and hide this change, as the subclass code is identical. If a new subclass is added (without those calls) then we'll need to override those methods to throw.